src/PowerShell/Help/Microsoft.WinGet.Client/Install-WinGetPackage.md

```diff
@@ -0,0 +1,548 @@
+
+```yaml
+Type: System.String
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -Location
+
+Specify the file path where you want the packed to be installed. The installer must be able to
+support alternate install locations.
+
+```yaml
+Type: System.String
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -Log
+
+Specify the location for the installer log. The value can be a fully-qualified or relative path and must include the file name. For example: `$env:TEMP\package.log`.
+
+```yaml
+Type: System.String
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -MatchOption
+
+Specify the match option for a WinGet package query. This parameter accepts the following values:
+- `Equals`
+- `EqualsCaseInsensitive`
+- `StartsWithCaseInsensitive`
+- `ContainsCaseInsensitive`
+
+```yaml
+Type: Microsoft.WinGet.Client.PSObjects.PSPackageFieldMatchOption
+Parameter Sets: FoundSet
+Aliases:
+Accepted values: Equals, EqualsCaseInsensitive, StartsWithCaseInsensitive, ContainsCaseInsensitive
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -Mode
+
+Specify the output mode for the installer. The parameter accepts the following values:
+- `Default`
+- `Silent`
+- `Interactive`
+
+```yaml
+Type: Microsoft.WinGet.Client.PSObjects.PSPackageInstallMode
+Parameter Sets: (All)
+Aliases:
+Accepted values: Default, Silent, Interactive
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -Moniker
+
+Specify the moniker of the WinGet package to install. For example, the moniker for the
+Microsoft.PowerShell package is `pwsh`.
+
+```yaml
+Type: System.String
+Parameter Sets: FoundSet
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -Name
+
+Specify the name of the package to be installed.
+
+```yaml
+Type: System.String
+Parameter Sets: FoundSet
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -Override
+
+Specify arguments to pass to the installer as a single string. To pass multiple arguments, include
+the arguments in the string. The arguments must be provided in the format expected by the installer.
+If the string contains spaces, it must be enclosed in quotes. This string overrides the arguments in
+the package manifest.
+
```

Comment:
Should we clarify the difference with `-Custom`?

Reply:
Updated description for both -Custom and -Override to clarify the difference